rhombus/tests/check.rhm

```diff
@@ -0,0 +1,35 @@
+#lang rhombus
+
+// I used the check to check the check…
+check:
+  check: values(1, [2, 3])
+         ~matches values(a, [b, c])
+  ~completes
+
+macro 'with_err_to_string:
+         $body ...':
+  'block:
+     def new_port = Port.open_output_string()
+     parameterize {Port.current_error : new_port}:
```

Comment:
(Sorry that I’m being picky here.)  I notice that Matthew favors putting spaces after opening and before ending braces in `parameterize`, so maybe sticking to that convention is better:

```
parameterize { some_param: some_value }:
  ....
````